bfscripts.spec.rpkg

```diff
@@ -0,0 +1,56 @@
+%global debug_package %{nil}
```

Comment:
Question: are you replacing the bfscripts.spec with bfscripts.spec.rpkg ?

Reply:
That's the plan. bfscripts.spec was auto-generated by Yocto, and as far as I know didn't even work properly outside of that context. Does anybody use the .spec in this repository?